cudapoa/src/cudapoa_batch.hpp

```diff
@@ -547,6 +548,12 @@ class CudapoaBatch : public Batch
 
         if (scores_size > avail_scorebuf_mem_)
         {
+            if (get_total_poas() == 0)
+            {
+                std::cout << "Memory available " << std::fixed << std::setprecision(2) << ((double)avail_scorebuf_mem_) / 1024. / 1024. / 1024.;
```

Comment:
`static_cast` for casting `avail_scorebuf_mem`

Reply:
good catch, it was an oversight due to copy/paste from an older code I had. fixed.